### ISSUE TITLE
[Bug] [ ERROR ]  Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.middle.PartialInfer.PartialInfer'>): Stopped shape/value propagation at "StatefulPartitionedCall/model_1/stem_conv/Conv2D" node.

### ISSUE THREAD

**Issue by MiraGaikwad** [openvinotoolkit/openvino#12742]
##### System information (version)
- OpenVINO => 2021.4.2
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2017
- Problem classification: Model Conversion
- Framework: TensorFlow 2.8.0
- Model name: EfficientNetV2S

##### Detailed description
1. I have trained the model in google Colab and saved file in h5 format.
2. I then frozen that model.
3. Now trying to convert that mode into IR format 
 but I am facing below error - 

[ ERROR ]  Cannot infer shapes or values for node "StatefulPartitionedCall/model_1/stem_conv/Conv2D".
[ ERROR ]  index 3 is out of bounds for axis 0 with size 3
[ ERROR ]
[ ERROR ]  It can happen due to bug in custom shape infer function <function Convolution.infer at 0x000001E97F619D08>.
[ ERROR ]  Or because the node inputs have incorrect values/shapes.
[ ERROR ]  Or because input shapes are incorrect (embedded to the model or passed via --input_shape).
[ ERROR ]  Run Model Optimizer with --log_level=DEBUG for more information.
[ ERROR ]  Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.middle.PartialInfer.PartialInfer'>): Stopped shape/value propagation at "StatefulPartitionedCall/model_1/stem_conv/Conv2D" node.

##### Steps to reproduce

Please find the H5 file can be found in attachment

https://drive.google.com/file/d/15Js2rughGyaAI55SGk0sS0iLKoHotb29/view?usp=sharing

1. I am using below code to convert model into .pb format. 
import tensorflow as tf

model = tf.keras.models.load_model('EfficientNetV2S_New_baseline.h5')
tf.saved_model.save(model,'saved_model')

2. Using below command to convert model into IR format where I am facing above error

mo --saved_model_dir saved_model --data_type FP16 --output_dir IR_model_FP16 --input_shape [1,224,224,3]

Could you please help over this ?

-->

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [ ] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [ ] There is reproducer code and related data files: images, videos, models, etc.

**Comment by Iffa-Meah** [comment]
@MiraGaikwad I had requested access to your model file.

![image](https://user-images.githubusercontent.com/79643476/186555149-ffe1d6a5-5290-44c8-aaa1-d51171ec67db.png)

**Comment by MiraGaikwad** [comment]
@Iffa-Meah - I have given the access to the model. Please let me know if you are able to access it.

Thank you..!!
Mira

**Comment by Iffa-Meah** [comment]
@MiraGaikwad 
I converted your h5 model into Saved Model in Google Colab:

<img width="685" alt="gcolab" src="https://user-images.githubusercontent.com/79643476/186828044-330c551b-7deb-40d6-9e2e-08f59c3a9035.png">

I used OpenVINO Model Optimizer to get IR format:

<img width="554" alt="mo1" src="https://user-images.githubusercontent.com/79643476/186828106-ed33944f-3d65-4260-ac7e-6a2e183cdc69.png">


From my end it seems that you need to provide the --input for --input_shape.
The --input is a quoted list of comma-separated input nodes names with shapes, data types, and values for freezing.

You may refer to the [OpenVINO MO general (framework-agnostic) parameters](https://docs.openvino.ai/2020.2/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html#when_to_reverse_input_channels)

**Comment by MiraGaikwad** [comment]
@Iffa-Meah - I tried using the option --input and it gave me below error -

![image](https://user-images.githubusercontent.com/84037638/186929576-286b8ed9-b3d1-4536-b100-c60659073afa.png)

I tried multiple other options but no luck yet. 

such as -
mo --saved_model_dir saved_model_2.8.2 --data_type FP16 --output_dir IR_model_FP16 --input [1 224 224 3]
mo --saved_model_dir saved_model_2.8.2 --data_type FP16 --output_dir IR_model_FP16 --input "[1 224 224 3]"
mo --saved_model_dir saved_model_2.8.2 --data_type FP16 --output_dir IR_model_FP16 --input "data[1 224 224 3]"
mo --saved_model_dir saved_model_2.8.2 --data_type FP16 --output_dir IR_model_FP16 --input_shape [1,224,224,3] --input "data[1 224 224 3]"

Could you please let me know the process you followed or exact command you used ?

**Comment by Iffa-Meah** [comment]
@MiraGaikwad this depends on how you named your inputs.
For example : 
name of input: input1, shape: [16,80,300]) 
name of output: output1 ,  shape: [16,576,13])

Conversion command:
mo --saved_model_dir savedmodel --data_type FP16 --input input1 --output output1
mo  --saved_model_dir savedmodel --data_type FP16 --input_shape [16,80,300]

**Comment by Iffa-Meah** [comment]
Closing issue, feel free to re-open or start a new issue if additional assistance is needed.

**Comment by MiraGaikwad** [comment]
This is still not working for me and giving me the same error.

Could you please help on this ? I was able to convert for other models in last month but it suddenly stopped working.

If you are able to convert then is is possible to share the IR files with me for the model which I have shared with you. I can use them further for inference.

`[ ERROR ]  Cannot infer shapes or values for node "StatefulPartitionedCall/model_1/block1a_project_conv/Conv2D".
[ ERROR ]  index 3 is out of bounds for axis 0 with size 0
[ ERROR ]
[ ERROR ]  It can happen due to bug in custom shape infer function <function Convolution.infer at 0x000001AF0BF5F598>.
[ ERROR ]  Or because the node inputs have incorrect values/shapes.
[ ERROR ]  Or because input shapes are incorrect (embedded to the model or passed via --input_shape).
[ ERROR ]  Run Model Optimizer with --log_level=DEBUG for more information.
[ ERROR ]  Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.middle.PartialInfer.PartialInfer'>): Stopped shape/value propagation at "StatefulPartitionedCall/model_1/block1a_project_conv/Conv2D" node.
 For more information please refer to Model Optimizer FAQ, question #38. (https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_Model_Optimizer_FAQ.html?question=38#question-38)`

**Comment by Iffa-Meah** [comment]
The latest comment which relates to this new inquiry is discussed in the GitHub case [#12884](https://github.com/openvinotoolkit/openvino/issues/12884).
